jwst/assign_wcs/nirspec.py

```diff
@@ -1224,8 +1224,9 @@ def nrs_ifu_wcs(input_model):
                       'nrs_focus': imaging,
                       'nrs_mimf': imaging,
                       'nrs_bota': imaging,
-                      'nrs_autoflat': not_implemented_mode, #TBD
-                      'nrs_autowave': not_implemented_mode, #TBD
+                      'nrs_autoflat': not_implemented_mode,
+                      'nrs_autowave': not_implemented_mode,
                       'nrs_lamp': slits_wcs,
-                      'nrs_brightobj': slits_wcs
+                      'nrs_brightobj': slits_wcs,
+                      'nrs_dark': not_implemented_mode,
```

Comment:
This list appears complete. I would think that NRS_AUTOWAVE could probably go through WCS assignment, but the problem is that it's used for slits and ifu, so which one would you assign it to in this list?